packages/flutter/lib/src/widgets/form.dart

```diff
@@ -68,34 +74,41 @@ class FormState extends State<Form> {
 
   @override
   Widget build(BuildContext context) {
+    if (config.autovalidate)
+      _validate();
     return new _FormScope(
       formState: this,
       generation: _generation,
       child: config.child
     );
   }
 
-  /// Saves every FormField that is a descendant of this Form.
+  /// Saves every [FormField] that is a descendant of this [Form].
```

Comment:
I mean "descendant that is not a descendant of a deeper Form" but that's a mouthful. FormFields will register themselves with the closest ancestor Form.